Maps/Perth/PH_TCU.xml

```diff
@@ -334,10 +314,108 @@
       <Point Name="WOODMAN PT">WMP</Point>
       <Point Name="WUNDOWIE">WUN</Point>
       <Point Name="WUNGONG RESERVOIR">WUNG</Point>
-      <Point Name="YANGEBUP LAKE">YGB</Point>
       <Point Name="YORK TOWNSHIP">ORKT</Point>
     </Label>
   </Map>
+  <Map Type="System2" Name="YPJT_ARRS_DEPS" Priority="1" Center="-32.098611+115.8808">
+    <!--Powerhouse/Boatyard arival-->
+    <Line Pattern="Broken">
+      POWR/ADWD
+    </Line>
+    <Line Pattern="Broken">
+      BOAT/ADWD 
+    </Line>
+    <Label LabelOrientation="NW" Alignment="Right">
+      <Point Name="POWER HOUSE ARIVAL">POWR</Point>
+      <Point Name="BOATYARD ARIVAL">BOAT</Point>
+    </Label>
+    <Label LabelOrientation="N">
+      <Point>POWR</Point>
+      <Point>ADWD</Point>
+      <Point>BOAT</Point>
+    </Label>
+    <Symbol Type="HollowSquare">
+     <Point>POWR</Point>
+     <Point>ADWD</Point>
+     <Point>BOAT</Point>
+    </Symbol>
+    <!--Yangebup Departure-->
+    <Line Pattern="Hash">
+    YGB/TOML
+    </Line>
+    <Label LabelOrientation="SE">
+      <Point Name="YANGEBUP DEPARTURE">YGB</Point>
+    </Label>
+    <Label LabelOrientation="NW">
+      <Point Name="YGB">YGB</Point>
+      <Point Name="TOML">TOML</Point>
+    </Label>
+    <Symbol Type="HollowSquare">
+      <Point>YGB</Point>
+      <Point>TOML</Point>
+    </Symbol>
+    <!--Russle and Oakford Arivals-->
+    <Line Pattern="Broken">
+      OAKF/FDL/-32.138888888889+115.91361111111
+    </Line>
+    <Line Pattern="Broken">
+      RUSS/FDL
+    </Line>
+    <Label LabelOrientation="SE">
+      <Point Name="OAKFORD ARIVAL">OAKF</Point>
+      <Point Name="RUSSELL ARIVAL">RUSS</Point>
+    </Label>
+    <Label LabelOrientation="NW">
+      <Point Name="RUSS">RUSS</Point>
+      <Point Name="OAKF">OAKF</Point>
+      <Point Name="FDL">FDL</Point>
+    </Label>
+    <Symbol Type="HollowSquare">
+     <Point>FDL</Point>
+     <Point>OAKF</Point>
+     <Point>RUSS</Point>
+    </Symbol>
+    <!--Shops Departure-->
+    <Line Pattern="Hash">
+    -32.111944444444+115.93861111111/SHOP
+    </Line>
+    <Label LabelOrientation="NE">
+      <Point Name="SHOPS DEPARTURE">-32.111944444444+115.93861111111</Point>
+    </Label>
+    <Label LabelOrientation="NW">
+      <Point Name="SHOP">SHOP</Point>
+    </Label>
+    <Symbol Type="HollowSquare">
+      <Point>SHOP</Point>
+    </Symbol>
+    <!--Fremantle Golfcourse Departure-->
+    <Line Pattern="Hash">
+    -32.07036705892901+115.84903482859521/FREM
+    </Line>
+    <Label LabelOrientation="SE">
+      <Point Name="FREEMANTLE DEPARTURE">-32.07036705892901+115.84903482859521</Point>

```

Comment:
Could you please fix the spelling of **Fremantle** :)

Reply:
Will push to fix